frontend/packages/ux-editor/src/components/config/componentSpecificContent/AttachmentList/AttachmentListContent.tsx

```diff
@@ -40,10 +40,10 @@ export const AttachmentListContent = ({
         value={selectedDataTypes}
         onValueChange={onChange}
       >
-        {currentAvailableAttachments?.map((attachment) => {
+        {currentAvailableAttachments?.map((attachment, index) => {
           return (
             <Combobox.Option
-              key={attachment}
+              key={index}
```

Comment:
That's a test, it wasn't meant to be committed as using `index` for the `key` is not recommended

Reply:
hehe, will change it back :)